modules/openapi-generator/src/main/resources/typescript-fetch/package.mustache

```diff
@@ -2,27 +2,15 @@
   "name": "{{npmName}}",
   "version": "{{npmVersion}}",
   "description": "OpenAPI client for {{npmName}}",
-  "author": "OpenAPI-Generator Contributors",
-  "keywords": [
-    "fetch",
-    "typescript",
-    "openapi-client",
-    "openapi-generator",
-    "{{npmName}}"
-  ],
-  "license": "Unlicense",
+  "author": "OpenAPI-Generator",
   "main": "./dist/index.js",
   "typings": "./dist/index.d.ts",
   "scripts" : {
     "build": "tsc --outDir dist/",
     "prepublishOnly": "npm run build"
   },
-  "dependencies": {
-    "portable-fetch": "^3.0.0"
-  },
   "devDependencies": {
-    "@types/node": "^8.0.9",
-    "typescript": "^2.4"
+    "typescript": "^2.9"
```

Comment:
is typescript 2.9 really required?

Reply:
good question. probably not. i'll revert it.